doc/api/https.md

```diff
@@ -46,7 +46,7 @@ added: v8.0.0
 
 See [`http.Server#keepAliveTimeout`][].
 
-## https.createServer(options[, requestListener])
+## https.createServer([options][, requestListener])
```

Comment:
I don't think we have to change this in the documentation, since `tls.Server()` will still require a configuration (a certificate and key at minimum). The benefit of guarding early before `tls.Server()` is called is to avoid strange JavaScript runtime errors.